Automattic/jetpack
# Sharing block: remove check for existing block in post content

## Proposed changes:

This is no longer necessary after some improvements to the Block Hooks API.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Start with a site that's connected to WordPress.com, using a block-based theme, and where the Sharing module is **not** active.
* Enable auto-addition of the sharing block to single post templates: `wp option update jetpack_sharing_buttons_auto_add 1`
* Visit a single post ; you should see sharing buttons being added at the bottom of the post.
* Now go to Appearance > Editor > Templates > Single Posts
* Add a sharing block and configure it, right below the post title for example.
* Remove any sharing block you may see at the bottom of the post content.
* Save your changes.
* Now visit a single post again ; you should only see the new sharing block you added to the top of the template, under the post title.
